components/controls/BoxedTabs.tsx

```diff
@@ -43,13 +48,11 @@ const StyledTab = styled.button<{ active: boolean }>`
   margin-bottom: -1px;
   min-width: 128px;
   min-height: 56px;
-  cursor: pointer;
+  ${props => !props.active && 'cursor: pointer;'}
   outline: 0;
   padding: calc(2 * ${props => props.theme.sizes.gridSize});
 
-  &:hover {
-    background-color: ${props => props.theme.colors.barBackgroundColorHighlight};
-  }
+  ${props => (!props.active ? highlightHoverMixin(props.theme) : null)}
```

Comment:
I think you can avoid passing the `props.theme` and assume in the mixin that the parameters is `props: { theme: ThemedProps }`.